CHANGELOG.md

```diff
@@ -3,6 +3,177 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.34](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.33...v4.0.0-alpha.34)
+
+_July 21, 2021_
+
+Big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Add \"is empty\" and \"is not empty\" operators (#1997) @m4theushw
+- 💅 Improve the editing API (#1955) @m4theushw
+- 🐛 We have fixed the `scrollToIndexes` behavior (#2162) @oliviertassinari
```

Comment:
It's not great yet, it's still completely broken when we scroll enough done, and it's desorienting when going backwards.

```suggestion
- 🐛 We have improved the scroll keyboard (#2162) @oliviertassinari
```